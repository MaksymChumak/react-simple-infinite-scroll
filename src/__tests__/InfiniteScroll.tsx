import * as React from 'react';
import * as renderer from 'react-test-renderer';

import { InfiniteScroll } from '../.';

describe('<InfiniteScroll />', () => {

  it('renders InfiniteScroll correctly', () => {
    const tree = renderer.create(
      <InfiniteScroll
        isLoading={true}
        throttle={100}
        threshold={300}
        hasMore={true}
        onLoadMore={() => console.log()}
      >
        <div>Hello</div>
      </InfiniteScroll>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
