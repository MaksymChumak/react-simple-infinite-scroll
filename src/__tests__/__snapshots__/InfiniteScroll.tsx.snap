// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InfiniteScroll /> renders InfiniteScroll correctly 1`] = `
<div>
  <div>
    Hello
  </div>
  <div />
</div>
`;
